pwndbg/lib/memoize.py

```diff
@@ -6,18 +6,12 @@
 
 import functools
 import sys
+from collections.abc import Hashable
 from typing import Any
 from typing import Callable
 from typing import Dict
 from typing import List
 
-try:
-    # Python >= 3.10
-    from collections.abc import Hashable
-except ImportError:
-    # Python < 3.10
-    from collections import Hashable  # type: ignore
-
```

Comment:
I see this was added in https://github.com/pwndbg/pwndbg/pull/960. But it seems to be working for me without it?

Reply:
Oh, I think I understand. The original fix could just have been `from collections.abc import Hashable`, as this should work for Python versions all the way to Python 3.3.